website/inertia/pages/register.tsx

```diff
@@ -0,0 +1,119 @@
+import { Button } from '~/components/ui/button'
+import { Card, CardContent, CardHeader, CardTitle, CardDescription } from '~/components/ui/card'
+import { Input } from '~/components/ui/input'
+import { Label } from '~/components/ui/label'
+import { useError } from '~/hooks/use_error'
+import { useForm } from '@inertiajs/react'
+import { cn } from '~/lib/utils'
+import AppLayout from '~/layouts/applayout'
+
+export default function Login() {
+  const oauthError = useError('oauth')
+
+  const { data, setData, post } = useForm({
+    email: '',
+    password: '',
+    confirmPassword: '',
```

Comment:
While testing, I consistently encountered an error indicating that the passwords did not match. Renaming the variable resolved the issue on my end. It might have been a mistake on my part, but it could be worth double-checking if the original variable name is causing unintended behavior.
```suggestion
    password_confirmation: '',
```

Reply:
Thank you!